csharp/src/Google.Protobuf/Google.Protobuf.nuspec

```diff
@@ -24,7 +24,7 @@
       <group targetFramework="monotouch" />
       <group targetFramework="monoandroid" />
       <!-- Dependencies for newer, more granular platforms (.NET Core etc) -->
-      <group targetFramework="dotnet">
+      <group targetFramework="netstandard1.0">
         <dependency id="System.Collections" version="4.0.0" />
```

Comment:
I think I'd prefer to use the newest refgen tool - from what I understand of how dependencies work, having "trimmed" dependencies feels like a better way forward here. (I don't quite understand why netstandard.library 1.6.0 is suggested instead of 1.0, when I'm targeting 1.0, but that's a different matter...)

Will try the beta of refgen now. (I'm not sure it's fully operational in this project anyway at the moment...)